xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Videos/WebHome.xml

```diff
@@ -98,9 +140,16 @@
       &lt;div class="videoCard-title"&gt;
         $escapetool.xml($data.title)
       &lt;/div&gt;
+      &lt;p class='sr-only'&gt;
+        $escapetool.xml($services.localization.render('help.videos.hint'))
+      &lt;/p&gt;
       &lt;ul&gt;
         #foreach ($topic in $data.topics)
-          &lt;li&gt;$topic&lt;/li&gt;
+          &lt;li&gt;
+            &lt;a href="$topic.url"&gt;
+              $topic.label
```

Comment:
I think there should be HTML escaping here.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2466/commits/5a5e8ebed90bd22a5888290046ec60f69f664275 :+1: